elfpy/hyperdrive_interface/hyperdrive_interface.py

```diff
@@ -166,28 +167,26 @@ def get_hyperdrive_config(hyperdrive_contract: Contract) -> dict[str, Any]:
         The hyperdrive config.
     """
     hyperdrive_config: dict[str, Any] = eth.smart_contract_read(hyperdrive_contract, "getPoolConfig")
-    pool_config = {}
+    pool_config: dict[str, Any] = {}
     pool_config["contractAddress"] = hyperdrive_contract.address
-    pool_config["baseToken"] = hyperdrive_config.get("baseToken", None)
-    pool_config["initialSharePrice"] = eth.convert_scaled_value(hyperdrive_config.get("initialSharePrice", None))
-    pool_config["minimumShareReserves"] = eth.convert_scaled_value(hyperdrive_config.get("minimumShareReserves", None))
-    pool_config["positionDuration"] = hyperdrive_config.get("positionDuration", None)
-    pool_config["checkpointDuration"] = hyperdrive_config.get("checkpointDuration", None)
+    pool_config["baseToken"] = hyperdrive_config["baseToken"]
+    pool_config["initialSharePrice"] = FixedPoint(scaled_value=hyperdrive_config["initialSharePrice"])
+    pool_config["minimumShareReserves"] = FixedPoint(scaled_value=hyperdrive_config["minimumShareReserves"])
+    pool_config["positionDuration"] = hyperdrive_config["positionDuration"]
+    pool_config["checkpointDuration"] = hyperdrive_config["checkpointDuration"]
     # Ok so, the contracts store the time stretch constant in an inverted manner from the python.
     # In order to not break the world, we save the contract version as 'invTimeStretch' and invert
     # that to get the python version 'timeStretch'
-    pool_config["invTimeStretch"] = hyperdrive_config.get("timeStretch", None)
-    pool_config["timeStretch"] = (
-        FixedPoint(1) / FixedPoint(scaled_value=hyperdrive_config.get("timeStretch", FixedPoint(1)))
-    ).scaled_value
-    pool_config["governance"] = hyperdrive_config.get("governance", None)
-    pool_config["feeCollector"] = hyperdrive_config.get("feeCollector", None)
-    curve_fee, flat_fee, governance_fee = hyperdrive_config.get("fees", (None, None, None))
-    pool_config["curveFee"] = eth.convert_scaled_value(curve_fee)
-    pool_config["flatFee"] = eth.convert_scaled_value(flat_fee)
-    pool_config["governanceFee"] = eth.convert_scaled_value(governance_fee)
-    pool_config["oracleSize"] = eth.convert_scaled_value(hyperdrive_config.get("oracleSize", None))
-    pool_config["updateGap"] = hyperdrive_config.get("updateGap", None)
+    pool_config["invTimeStretch"] = hyperdrive_config["timeStretch"]
+    pool_config["timeStretch"] = FixedPoint(1) / FixedPoint(scaled_value=hyperdrive_config["timeStretch"])
```

Comment:
do we still want to store timeStretch inverted from how solidity does it? this could be a good time to change our convention to match theirs.